source/mechanism.md

```diff
@@ -115,14 +115,46 @@ The primary aim with this deployment was to provide the [dxDAO](https://dutchx.r
+- It is not necessary that the token is either listed for trading or actively traded
+- The auctioneer calls the function *updateApprovalOfToken* in the [Token Whitelist contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/base/TokenWhitelist.sol#L28)
+- There is no time lag to the execution of this function
+- It applies immediately and is effective upon users *claiming* their tokens of whitelisted tokens (if traded in a whitelisted pair), also if the auction had finished prior to triggering this function
+
 #### Changing the threshold to start auctions
-TODO
+Once this defined sellVolume (deposit) threshold is reached by both opposite auctions (each), and the prior auctions have finished, the new auctions will commence.
+- The auctioneer calls the function *updateThresholdNewAuction* of the [Master Contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchange.sol) (via the [Proxy](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchangeProxy.sol)
+- At the time of writing, this value is set at USD1000 (in uint USD)
+- There is no time lag to calling this function
+- It applies to all auctions (no individual change possible) that have not started (also if currently waiting for funding)
+
 #### Changing the threshold to add tokens to the DutchX protocol
-TODO
+- Once this defined sellVolume (deposit) threshold is reached, the first ever auction for this pair will commence.
+- The very first auction for an entirely new token on the DutchX has to be with WETH, where the defined threshold has to be reached solely on the WETH sellVolume.
+- For the very first aucion of another token pair (without WETH), the defined threshold can be reached combined on both sides of the auctions sellVolumes.
+- The auctioneer calls the function *updateThresholdNewTokenPair* of the [Master Contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchange.sol) (via the [Proxy](https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchangeProxy.sol)
+- At the time of writing this value is set at USD1000 (in uint USD)
+- There is no time lag to calling this function
+- It applies to all auctions (no individual change possible) that have not started (also if currently waiting for funding)
+
 #### Setting a new external ETH/USD price feed 
-TODO*(change only executed with a 30day time lag)*
+The DutchX needs an ETH-USD price feed for two purposes: (i) to calculate the thresholds mentioned above as well as (ii) calculating the liquidity contribution which can be settled in USD. 
+- The auctioneer calls the function *initiateEthUsdOracleUpdate* in the [EthOracle Contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/base/EthOracle.sol#L20)
+- There is a 30day time lag for execution of this function 
+- After the 30days are up, the auctioneer calls the function *updateEthUSDOracle* in the same contract
```

Comment:
In the next sentence you correctly say that anyone can call `updateEthUSDOracle`
Maybe change this one to `the auctioneer or anyone else`